.github/workflows/check_license.yml

```diff
@@ -6,6 +6,11 @@ on:
 jobs:
   check-spdx-headers:
     runs-on: ubuntu-latest
+
+    concurrency:
+      group: ${{ github.ref }}-${{ github.workflow }}-check-spdx-headers
```

Comment:
General idea is to have a unique group-id for each job. I first tried including `${{ github.job }}` but that is not set when this is evaluated, so now the pattern is ref-workflow-<job title>